ci/run_tests_test.go

```diff
@@ -60,3 +65,144 @@ func TestRunTests(t *testing.T) {
 	}
 	t.Logf("\n%s\nExecTime: %v\nSecret: %v\n", ed.out, ed.execTime, info.RandomSecret)
 }
+
+func TestRecordResults(t *testing.T) {
+	f, err := ioutil.TempFile(os.TempDir(), "testing")
```

Comment:
Creating a new test database can now be done with a new internal function from #479:
```golang
db, cleanup := internal.TestDB(t)
defer cleanup()
```

Reply:
I'll update it when #479 is merged into master.

Reply:
Merged #479

Reply:
Updated.